gooddata-sdk/gooddata_sdk/compute/service.py

```diff
@@ -40,8 +43,19 @@ def retrieve_result_cache_metadata(self, workspace_id: str, result_id: str) -> R
         :param result_id: execution result ID
         :return: execution result's metadata
         """
-        return ResultCacheMetadata(
-            result_cache_metadata=self._actions_api.retrieve_execution_metadata(
-                workspace_id, result_id, _check_return_type=False
-            )
+        result_cache_metadata, _, http_headers = self._actions_api.retrieve_execution_metadata(
+            workspace_id,
+            result_id,
+            _check_return_type=False,
+            _return_http_data_only=False,
         )
+        custom_headers = self._api_client.custom_headers
+        if "X-GDC-TRACE-ID" in custom_headers and "X-GDC-TRACE-ID" in http_headers:
+            logging.getLogger(__name__).info(
```

Comment:
The same comment as above. Move `logging.getLogger(__name__)` to package variable.